build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -171,7 +171,9 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
         {
             var hostVersion = c.BuildContext.Get<HostVersion>("HostVersion");
             var configuration = c.BuildContext.Get<string>("Configuration");
-            string rid = c.BuildContext.Get<string>("TargetRID");
+            string rid = DotNetCli.Stage0.GetRuntimeId();
```

Comment:
Please consider the cross build situation. When I tried to build for ubuntu arm, I passed the TargetRID value through --env-vars like below.
```
$ ./build.sh  --env-vars DISABLE_CROSSGEN=1,TARGETPLATFORM=arm,TARGETRID=ubuntu.14.04-arm,CROSS=1,ROOTFS_DIR=/home/jyoung/git/dotnet/rootfs-coreclr/arm/
```